src/core/library.c

```diff
@@ -889,6 +889,47 @@ QuicLibrarySetGlobalParam(
 
         break;
 
+    case QUIC_PARAM_GLOBAL_RAW_DATAPATH_PROCS:
+        if (Buffer == NULL) {
+            Status = QUIC_STATUS_INVALID_PARAMETER;
+            break;
+        }
+
+        if (BufferLength < sizeof(uint32_t) || BufferLength % sizeof(uint32_t) != 0) {
```

Comment:
We should support setting null/zero-length to clear.

Reply:
Done